.github/workflows/coding-standard.yml

```
@@ -1,9 +1,52 @@
-name: Coding Standard
+name: M2 Coding Testing
```

```suggestion
name: Coding Standard
```

I think "Coding Standard" or "M2 Coding Standard" is the better choice here.